Utilities/AppleEfiSignTool/AppleEfiSignTool.c

```diff
@@ -208,7 +227,7 @@ LLVMFuzzerTestOneInput (
       CopyMem (Copy, Data, Size);
 
       NewSize = (UINT32)Size;
-      PeCoffVerifyAppleSignature (Copy, &NewSize);
+      PeCoffVerifyAppleSignature (Copy, &NewSize, FALSE);
```

Comment:
Here and just above, if commented in, needs more work to cover new code path accessed with TRUE.

Reply:
On reflection, the path accessed with TRUE is not enabled except in userspace builds, so this is probably still enough to test code that will be enabled in production.